frontend/src/metabase/static-viz/components/XYChart/XYChart.tsx

```diff
@@ -310,6 +219,69 @@ export const XYChart = ({
         lineHeight={style.legend.lineHeight}
         fontSize={style.legend.fontSize}
       />
+
+      <Group top={margin.top} left={xMin}>
```

Comment:
Is there a way to add a padding to the domain so that labels do not overlap with Y-axes?
<img width="583" alt="Screen Shot 2022-08-30 at 2 09 13 AM" src="https://user-images.githubusercontent.com/14301985/187308107-f0ab6b51-6566-4a69-9b4d-363e95d6fb97.png">

Reply:
@alxnddr Possibly. It was annoying too, but since everything else that's more important got taken care of, I can start taking a look at this. If this isn't too hard or complicated, I'll try to make it into this PR.

Reply:
@alxnddr Addressed in https://github.com/metabase/metabase/pull/24938/commits/18ffd85122d238424bc0b90aa4ffa95a1c73752e